fix bad English, broken links, s/CHE/Che/g...

nickboldt opened the pull request:
fix bad English, broken links, s/CHE/Che/g s/openshift/OpenShift/g, etc. in readme

Also update doc so we're not talking about the "Che 7 Theia stack" which was a beta feature in Che 6 and is now the default IDE in Che 7. 

Change-Id: I74b281dc668cb8ca5595b3d5a57ea5ac1bcfab69
Signed-off-by: nickboldt <nboldt@redhat.com>

skabashnyuk commented on README.md:
```diff
@@ -76,23 +67,23 @@ $ minishift start --memory=8GB
 > Note: in case if you delete minishift virtual machine(`minishift delete`) and create it again, this url will be changed.
 
-Register new user on the CHE_INFRA_KUBERNETES_MASTER__URL page.
+Register new user on the Che_INFRA_KUBERNETES_MASTER__URL page.
```
`Che_INFRA_KUBERNETES_MASTER__URL`- this might be incorrect

skabashnyuk commented on README.md:
```diff
@@ -76,23 +67,23 @@ $ minishift start --memory=8GB
 
 From this command output you need:
- - Minishift master url. In this case it's `https://192.168.42.159:8443`. Let's call it 'CHE_INFRA_KUBERNETES_MASTER__URL'. We can store this variable in the terminal session to use it for next commands:
+ - Minishift master url. In this case it's `https://192.168.42.159:8443`. Let's call it 'Che_INFRA_KUBERNETES_MASTER__URL'. We can store this variable in the terminal session to use it for next commands:
```
Che_INFRA_KUBERNETES_MASTER__URL - is this variable intentionally have `Che_` instead of `CHE_`

MichalMaler commented on README.md:
```diff
@@ -17,33 +18,23 @@ Build docker image with che-machine-exec manually:
+First, [build Eclipse Che Assembly](#build-eclipse-che-assembly).
 
- Run ocp.sh with arguments:
+To deploy Eclipse Che to OpenShift you can use [these templates](https://github.com/eclipse/che/blob/master/deploy/openshift/).
```
To deploy Eclipse Che to OpenShift, use [these templates]

MichalMaler commented on README.md:
```diff
@@ -2,12 +2,13 @@
-CHE machine exec uses json-rpc protocol to communication with client.
+Used to spawn terminals or command processes.
+
+Che machine exec uses json-rpc protocol to communicate with client.
```
Che machine exec uses json-rpc protocol to communicate with a client.

MichalMaler commented on README.md:
````diff
@@ -152,51 +142,54 @@ kubectl config use-context minikube
 ```
 
-For test purpose it's not nessure build all Eclipse CHE, build 'assembly-main' maven module is pretty enough:
+You can save time by simply building the `assembly-main` module, rather than the whole Eclipse Che project.
````
Save the time by building the `assembly-main` module, instead of the whole Eclipse Che project.

skabashnyuk commented:
@nickboldt are you sure you want environment variables to look like this `Che_INFRA_KUBERNETES_MASTER__URL` instead of `CHE_INFRA_KUBERNETES_MASTER__URL `?

sleshchenko commented on README.md:
```diff
@@ -17,33 +18,23 @@ Build docker image with che-machine-exec manually:
-To test che-machine-exec you need deploy Eclipse CHE to the openshift locally. [Prepare Eclipse CHE to deploy](#prepare-eclipse-che-to-deploy)
-
-To deploy Eclipse CHE to the local running openshift you can use [ocp.sh sript](https://github.com/eclipse/che/blob/master/deploy/openshift/ocp.sh).
+## Test che-machine-exec on Openshift
```
```suggestion
## Test che-machine-exec on OpenShift
```

nickboldt commented:
> @nickboldt are you sure you want environment variables to look like this Che_INFRA_KUBERNETES_MASTER__URL instead of CHE_INFRA_KUBERNETES_MASTER__URL ?

Good catch, that's why I asked for a PR review from the SMEs. hat was a global regex replacement that was too aggressive.

nickboldt commented:
@rkratky thanks for the review & PR on my PR. I took your changes, fixed a few things (incorrect code / 404'd code), updated to oc 3.11.latest, fixed the CHE_ variables, and then tried to fix the md bullet indents... with some success. Frankly, I'd be happier if we move to adoc but but is good enough for now.
